test/controller/bankrupt-officer.spec.ts

```diff
@@ -2,17 +2,21 @@ import { NextFunction, Request, Response } from 'express';
 import chai, { expect } from 'chai';
 import sinonChai from "sinon-chai";
 import sinon from "sinon";
-import axios from 'axios';
 
+import { BadosService } from 'private-api-sdk-node/dist/services/bankrupt-officer';
 import { bankruptOfficer } from "../../src/controller";
-import { logger, formattingOfficersInfo } from '../../src/utils';
+import { logger } from '../../src/utils';
 
 import { 
-  mockAxiosResponse,
+  mockGetResponse,
```

Comment:
why did we moved from mocking Axios to our mocking the custom Get response?

Reply:
We are no longer using axios for the request/response as we are using the SDK instead (SDK API client)